### PR TITLE
Use more semantic `required` class for label star

### PR DESCRIPTION
**Pull request by bluerogue251**
/cc @tomrecap

### CODE REVIEW

**Comment by hjast** [comment]
:+1:

File: src/styles/shared/modules/forms.scss [hunk 1]
```diff
@@ -12,6 +12,12 @@ form, .form-fields { // .form-fields is for Angular forms where it's impossible
   .requirement-star:after {
```
Comment (tomrecap) [inline src/styles/shared/modules/forms.scss]: Why not just change this line to `.requirement-star:after, .required:after {`?

Comment (bluerogue251) [inline src/styles/shared/modules/forms.scss]: Because lots of form elements have the `required` class, but I only want it only labels.

Comment (bluerogue251) [inline src/styles/shared/modules/forms.scss]: Oh, would it work to write this?

```
.requirement-star:after, label.required:after {
```

Comment (bluerogue251) [inline src/styles/shared/modules/forms.scss]: That doesn't seem to work for me.  ? :-(

**Pull request merged**